### PR TITLE
Federation release-1.4 images are pushed to a repository in its own project, use that instead of using the HEAD images.

### PR DESCRIPTION
**Pull request by madhusudancs**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/644)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 1]
```diff
@@ -691,7 +691,7 @@
                 export DNS_ZONE_NAME="k8s-federation-1-4.com."
                 export FEDERATIONS_DOMAIN_MAP="federation=k8s-federation-1-4.com."
                 export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the clusters will be created. Federation components are now deployed to the last one.
-                export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+                export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-f8n-1-4"
```
Comment (quinton-hoole) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Use $PROJECT instead?  It seems that would have avoided the bug in the first place?

**Comment by quinton-hoole** [comment]
LGTM

**Comment by madhusudancs** [comment]
Rolling back the $PROJECT substitution. It is causing travis to fail.

**Comment by madhusudancs** [comment]
Looks like the checks have passed for this commit. Merging it.

**Pull request merged**